Remove remaining links to `osu!_File_Formats\Db_(file_format)` (#6414)
ppy/osu-wiki: wiki/Beatmap/Online_offset/en.md

![online offset on web](img/online_offset.jpg)<!--TODO: add image of the new webpage once online offset gets added there-->

> All online offset values are stored locally in the [osu!.db](https://github.com/ppy/osu/wiki/Legacy-database-file-structure) file for later use (as `short`s). Storing online offset values in the osu!.db allows the values to be applied even if a player is playing offline as long as they have been connected to the internet prior to importing or playing the beatmap.

Reviewer: doesn't need to be mentioned. just remove the whole reference to the db file.

Reviewer 2: ```suggestion
All online offset values are stored locally for later use. This allows the values to be applied even if a player is playing offline as long as they have been connected to the internet prior to importing or playing the beatmap.
```
